internal/vault/vault.go

```diff
@@ -70,9 +71,11 @@ func FetchKeys(uid string) (*KeyPair, error) {
 	// specified type. In this case, we are filtering for fields of type
 	// "keyPair".
 	keys := records[0].GetFieldsByType("keyPair")[0]["value"].([]interface{})[0]
+	pass := records[0].GetFieldsByType("password")[0]["value"].([]interface{})[0].(string)
```

Comment:
If no passphrase is set in the record, this line fails and returns the below error to the user. Please add error handling for this use case.

```
error: panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/Keeper-Security/git-ssh-sign/internal/vault.FetchKeys({0xc000016210, 0x16})
        C:/Users/RickyWhite/Code/Keeper/git-ssh-sign/internal/vault/vault.go:74 +0x4ef
main.main()
        C:/Users/RickyWhite/Code/Keeper/git-ssh-sign/cmd/ssh-sign/main.go:74 +0x4e5

fatal: failed to write commit object
```